tests/unittests/test_dispatcher.py

```diff
@@ -20,7 +20,7 @@
     PYTHON_THREADPOOL_THREAD_COUNT,
     PYTHON_THREADPOOL_THREAD_COUNT_DEFAULT,
     PYTHON_THREADPOOL_THREAD_COUNT_MAX_37,
-    PYTHON_THREADPOOL_THREAD_COUNT_MIN,
+    PYTHON_THREADPOOL_THREAD_COUNT_MIN, HTTP_URI, REQUIRES_ROUTE_PARAMETERS,
```

Comment:
nit: run black for your changes to be consistent with style set in https://github.com/Azure/azure-functions-python-worker/pull/1526